docs/kubernetes/routegroup-crd.md

```diff
@@ -46,8 +46,7 @@ section of the reference.
 
 ## RouteGroup - top level object
 
-The route group spec can or must contain the hosts, backends, default backends and routes. Mandatory fields are
-the backends, and either the defaultBackends or the routes.
+The route group spec must contain hosts, backends, routes and optional default backends.
```

Comment:
https://github.com/zalando/skipper/blob/b193e5ff22137a2a4cb0a8cd28fc13d72ed73ea0/dataclients/kubernetes/deploy/apply/routegroups_crd.yaml#L35-L38

Reply:
Though I am confused:
https://github.com/zalando/skipper/blob/b193e5ff22137a2a4cb0a8cd28fc13d72ed73ea0/dataclients/kubernetes/routegroup.go#L520-L522
https://github.com/zalando/skipper/blob/b193e5ff22137a2a4cb0a8cd28fc13d72ed73ea0/dataclients/kubernetes/routegroup.go#L379-L381

Reply:
I think @aryszka did the implementation as documented and @aermakov-zalando changed the CRD that enforces within the apiserver that we won't get these (RG without hosts or without routes) cases.